go/client/cmd_simplefs_quota.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"fmt"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+	"golang.org/x/net/context"
+)
+
+// CmdSimpleFSQuota is the 'fs quota' command.
+type CmdSimpleFSQuota struct {
+	libkb.Contextified
+	git      bool
+	bytes    bool
+	archived bool
+}
+
+// NewCmdSimpleFSQuota creates a new cli.Command.
+func NewCmdSimpleFSQuota(
+	cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:  "quota",
+		Usage: "show quota usage for logged-in user",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(
+				&CmdSimpleFSQuota{Contextified: libkb.NewContextified(g)},
+				"quota", c)
+			cl.SetNoStandalone()
+		},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "git",
+				Usage: "show git usage",
+			},
+			cli.BoolFlag{
+				Name:  "bytes",
+				Usage: "show all data in bytes",
```

Comment:
Where is this used? We have `UsageBytes` and `LimitBytes`, but neither of them are guarded by a switch.

Reply:
It's used here to avoid humanizing the output: https://github.com/keybase/client/pull/12831/files#diff-ee8ff213e578cc065dedd53e8b3b08e9R72

Perhaps this help text is bad.